components/constants/brave_features.h

```diff
@@ -3,8 +3,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#ifndef BRAVE_COMMON_BRAVE_FEATURES_H_
-#define BRAVE_COMMON_BRAVE_FEATURES_H_
+#ifndef BRAVE_COMPONENTS_CONSTANTS_BRAVE_FEATURES_H_
+#define BRAVE_COMPONENTS_CONSTANTS_BRAVE_FEATURES_H_
```

Comment:
I think we should put `kBraveRewards` into appropriate place and remove this file.

Reply:
Done.